README.md

```diff
@@ -66,12 +66,15 @@ support for customizing or creating entirely new visual components.
 
 ### Fast results
 
-Flutter is fast. It's powered by the same hardware-accelerated 2D graphics
-library that underpins Chrome and Android: [Skia]. We architected Flutter to
+Flutter is fast. It's powered by hardware-accelerated 2D graphics
+libraries like [Skia] (that underpins Chrome and Android) and 
+[Impeller](/perf/impeller). We architected Flutter to
```

Comment:
Since this isn't on the website, I think you'll want to use the full link for this to work: https://docs.flutter.dev/perf/impeller

Also consider putting the link definition at the bottom of the readme to be consistent with the others.

Reply:
Ah, yes, done!